ui/addincident.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>iReporter</title>
+    <!-- include CSS -->
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Dancing+Script|Lato" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+        crossorigin="anonymous">
+</head>
+
+<body class="bg">
+    <nav class="navbar">
+        <div class="container">
+            <ul class="nav">
+                <li class="nav-item nav-brand">
+                    <a href="index.html">iReporter</a>
+                </li>
+            </ul>
+            <ul class="nav nav-right">
+                <li class="nav-item">
+                    <a href="./dashboard.html">Dashboard</a>
+                </li>
+                <li class="nav-item">
+                    <a href="./addincident.html">Create Incident</a>
+                </li>
+                <li class="nav-item">
+                    <a href="./incidences.html">My Incidences</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <div class="container">
+        <div class='incident-form-wrapper'>
+            <div>
+                <h1>Create an Incident</h1>
+            </div>
+            <form action='./incidences.html' enctype="multipart/form-data" class="incident-form">
+                <div class='form-group'>
+                    <label for=incident_type>Type  <span class='required'>*</span></label> 
+                   
+                    <select name="incident_type">
+                        <option value="">Select</option>
+                        <option value="red_flag">Red Flag</option>
+                        <option value="intervention">Intervention</option>
+                    </select>
+                </div>
+
+                <div class="form-group">
+                    <label for="title">Title  <span class='required'>*</span></label>
+                    <!-- <br> -->
+                    <input type="text" name="title" placeholder="2-5 word description">
+                </div>
+
+                <div class="form-group">
+                    <label for="description">Description <span class='required'>*</span></label>
+                    <!-- <br> -->
+                    <textarea name="description" placeholder="An explanation in detail of what you are reporting"></textarea>
+                </div>
+                <div class="form-group">
+                    <label for="media">Media</label>
+                    <input type="file" name="media" accept="video/*,image/*" mutliple>
```

Comment:
Typing error on 
<input type="file" name="media" accept="video/*,image/*" mutliple>
should be multiple